LeanKit-Labs/tag-release
# Cleaned up some unnecessary stuff that was happening in the pre-release workflow

### Short description of the work completed

> This PR cleans up some of the unnecessary stuff that was happening in the pre-release workflow. Should take care of #62;

### Steps to test (if not obvious)

1. Run `tag release --prerelease`...
2. Ensure that you are no longer prompted asking if you would like to update your log (this prompt has been removed)
3. Ensure that _ONLY_ the configured config file (typically `package.json` on a normal workflow...so that's what it will be in this case, unless you specifically told it to use a different config file) gets staged for commit before tagging.

### For Reviewer Use Only

  patch)
